### PR TITLE
Add viewport meta tag

### PR DESCRIPTION
**Pull request by mvano**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
       Push API
     </title>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width">
```
Comment (beverloo) [inline index.html]: This seems to be rather uncommon. Several other specifications seem to achieve resolution independence by setting "max-width: 50em" on <body>.

If you do decide to go this way, you should also specify initial-scale=1.0 (per http://mobile-html.appspot.com/#14)

Comment (mvano) [inline index.html]: Added initial-scale=1.0. I'll see about further styling later.

**Comment by beverloo** [comment]
SGTM

**Comment by mvano** [comment]
@efullea I think this is ready to be merged. Could you take a look please? Thanks!

**Pull request merged**